pkg/flytek8s/client.go

```diff
@@ -63,14 +55,29 @@ func GetRestClientConfig(kubeConfig, master string,
 		}
 		logger.Debugf(context.Background(), "successfully loaded kube config from %s", kubeConfig)
 	} else if k8sCluster != nil {
-		return GetRestClientConfigForCluster(*k8sCluster)
+		kubeConfiguration, err = RemoteClusterConfig(k8sCluster.Endpoint, k8sCluster.Auth)
+		if err != nil {
+			return nil, err
+		}
+		logger.Debugf(context.Background(), "successfully loaded kube configuration from %v", k8sCluster)
+
+		if k8sCluster.KubeClientConfig != nil {
+			kubeClientConfig = k8sCluster.KubeClientConfig
+		}
```

Comment:
Can we log here that we are using the rest config from remote cluster override

Reply:
added a message